lib/checkPullRequestJob.js

```diff
@@ -0,0 +1,210 @@
+const {SERVER_JOBS_ADMIN} = require('../userWhitelist.json');
+const REGISTRY_FILENAME = 'jobs_registry.py';
+const JOB_FILE_FIX = '_jobs_';
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem}
+ */
+const isJobFile = ({filename}) => {
+  return filename.includes(JOB_FILE_FIX);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem}
+ */
+const isNewFile = ({status}) => {
+  return status === 'added';
+};
+
+/**
+ * @param {string} filename
+ */
+const getJobNameFromFileName = (filename) => {
+  // Filename is the relative path to the file, we need to
+  // get the exact file name.
+  const substrings = filename.split('/');
+  const actualFileNameWithExtension = substrings[substrings.length - 1];
+  const extensionIndex = actualFileNameWithExtension.indexOf('.py');
+
+  return actualFileNameWithExtension.substring(0, extensionIndex);
```

Comment:
I am a bit confused on what this function is doing, are you obtaining the name of a job being added or the name of the file without any dir paths in it? Can you add some examples in comments and update the name of the function accordingly if it is being used to obtain file name without extensions.

Reply:
Can you add an example in the comments?